src/scenes/home/history/history.js

```diff
@@ -1,27 +1,25 @@
-import React, { Component } from 'react';
+import React from 'react';
 import Section from 'shared/components/section/section';
 import QuoteBanner from 'shared/components/quoteBanner/quoteBanner';
 import Timeline from './timeline/timeline';
 
-class History extends Component {
-  render() {
-    return (
-      <div>
-        <QuoteBanner
-          author="Collin Powell"
-          quote="There are no secrets to success. It is the result of preparation, hard work, learning from failure."
-        />
+function History() {
```

Comment:
To keep in-line with the rest of the codebase, would you please change this to ES6 syntax arrow function format?

`const = () => ( JSX CONTENT HERE );`

Reply:
Ok, will do it asap!